ViewModel/ProductListItem.php

```diff
@@ -70,7 +75,16 @@ public function getItemHtml(
             $this->cacheHelper->save($itemHtml, $productId);
         }
 
-        return sprintf('<esi:include src="/%s?product_id=%s" />', Cache::PRODUCT_CARD_PATH, $productId);
+        $storeId = $this->storeManager->getStore()->getId();
+        $customerGroupId = $this->customerSession->getCustomerGroupId();
+
+        return sprintf(
+            '<esi:include src="/%s?product_id=%s&store_id=%s&customer_group_id=%s&cardtype=default" />',
```

Comment:
`cardtype` could be removed. `default` is set in Helper/Cache.php:56 if `card_type` is not set in the esi:include

Reply:
Changes implemented